lib/tokenize.cpp

```diff
@@ -9226,7 +9229,7 @@ void Tokenizer::simplifyCppcheckAttribute()
 
 void Tokenizer::simplifyCPPAttribute()
 {
-    if (!isCPP() || mSettings.standards.cpp < Standards::CPP11)
```

Comment:
I don't see the point in not simplifying those attributes. Current compilers only show a warning even with `-std=c++03`.